webapp/components/admin_console/webhook_settings.jsx

```diff
@@ -141,7 +141,7 @@ export default class WebhookSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.iconTitle'
-                            defaultMessage='Enable Overriding Icon from Webhooks and Slash Commands: '
+                            defaultMessage='Enable integrations to override icons: '
```

Comment:
Propose "Enable webhooks and slash commands to override profile picture icons" // OAuth provider is being added to integrations and this setting won't apply. 

Also, we call it profile picture and icon in different places, propose use both words to make it clear.